target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8072-ax880.dts

```diff
@@ -0,0 +1,390 @@
+			};
+
+			partition@80000 {
+				label = "0:BOOTCONFIG1";
+				reg = <0x80000 0x20000>;
+				read-only;
+			};
+
+			partition@a0000 {
+				label = "0:QSEE_1";
+				reg = <0xa0000 0x180000>;
+				read-only;
+			};
+
+			partition@220000 {
+				label = "0:QSEE";
+				reg = <0x220000 0x180000>;
+				read-only;
+			};
+
+			partition@3a0000 {
+				label = "0:DEVCFG";
+				reg = <0x3a0000 0x10000>;
+				read-only;
+			};
+
+			partition@3b0000 {
+				label = "0:DEVCFG_1";
+				reg = <0x3b0000 0x10000>;
+				read-only;
+			};
+
+			partition@3c0000 {
+				label = "0:APDP";
+				reg = <0x3c0000 0x10000>;
+				read-only;
+			};
+
+			partition@3d0000 {
+				label = "0:APDP_1";
+				reg = <0x3d0000 0x10000>;
+				read-only;
+			};
+
+			partition@3e0000 {
+				label = "0:RPM_1";
+				reg = <0x3e0000 0x40000>;
+				read-only;
+			};
+
+			partition@420000 {
+				label = "0:RPM";
+				reg = <0x420000 0x40000>;
+				read-only;
+			};
+
+			partition@460000 {
+				label = "0:CDT_1";
+				reg = <0x460000 0x10000>;
+				read-only;
+			};
+
+			partition@470000 {
+				label = "0:CDT";
+				reg = <0x470000 0x10000>;
+				read-only;
+			};
+
+			partition@480000 {
+				label = "0:APPSBLENV";
+				reg = <0x480000 0x10000>;
+			};
+
+			partition@490000 {
+				label = "0:APPSBL_1";
+				reg = <0x490000 0xa0000>;
+				read-only;
+			};
+
+			partition@550000 {
+				label = "0:APPSBL";
+				reg = <0x530000 0xa0000>;
+				read-only;
+			};
+
+			partition@610000 {
+				label = "0:ART";
+				reg = <0x5d0000 0x40000>;
+				read-only;
+			};
+
+			partition@650000 {
+				label = "0:ETHPHYFW";
+				reg = <0x610000 0x80000>;
+				read-only;
+			};
+
+		};
+	};
+};
+
+//serial interface
+&blsp1_uart3 {
+	status = "okay";
+};
+
+&blsp1_uart5 {
+	status = "okay";
+};
+
+&crypto {
+	status = "okay";
+};
+
+&cryptobam {
+	status = "okay";
+};
+
+&prng {
+	status = "okay";
+};
+
+&qpic_bam {
+	status = "okay";
+};
+
+&qusb_phy_0 {
+	status = "okay";
+};
+
+&ssphy_0 {
+	status = "okay";
+};
+
+&usb_0 {
+	status = "okay";
+};
+
+&qpic_nand {
+	status = "okay";
+
+	nand@0 {
+		reg = <0>;
+		nand-ecc-strength = <4>;
+		nand-ecc-step-size = <512>;
+		nand-bus-width = <8>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "rootfs_1";
+				reg = <0x0000000 0x3400000>;
+			};
+
+			partition@3400000 {
+				label = "0:WIFIFW";
+				reg = <0x3400000 0x800000>;
+				read-only;
+			};
+
+			rootfs: partition@3c00000 {
+				label = "rootfs";
+				reg = <0x3c00000 0x3400000>;
+			};
+
+			partition@7000000 {
+				label = "0:WIFIFW_1";
+				reg = <0x7000000 0x800000>;
+				read-only;
+			};
+		};
+	};
+};
+
+&mdio {
+	status = "okay";
+
+	pinctrl-0 = <&mdio_pins>;
+	pinctrl-names = "default";
+   
+    ethernet-phy@0 {
+        reg = <0>;
+    };
+    ethernet-phy@1 {
+        reg = <1>;
+    };
+    ethernet-phy@2 {
+        reg = <2>;
+    };
+    ethernet-phy@3 {
+        reg = <3>;
+    };
```

Comment:
Please use tabs instead of spaces for indentation.

Reply:
Done.

Reply:
Not done, indentation is still broken.

Reply:
Also, do these PHY-s even exist as they are not being used and you specify only 2x QCA8081 in the specs?

Reply:
As robimarko wrote, you can remove ethernet-phy@[0-3]. The device has only 2 QCA8081 phys.
port@[0-3] properties in the switch node can be removed as well.

Reply:
> As robimarko wrote, you can remove ethernet-phy@[0-3]. The device has only 2 QCA8081 phys. port@[0-3] properties in the switch node can be removed as well.

removed.

Reply:
> Also, do these PHY-s even exist as they are not being used and you specify only 2x QCA8081 in the specs?

I hope to use these emtpy entries somehow in qca-ssdk. There is a spam messages related to these ports at the moment. Removed now.

Reply:
You mean these log entries?

```
phy_id = 0xffffffff phytype doesn't match
```
Those can be ignored, they should only occur during ssdk init.

Reply:
Yeah, those dont matter at all, its just QCA-s stupid expectation that 0-4 will always be present but its harmless.

Reply:
I'm talking about these log entries:
![image](https://github.com/openwrt/openwrt/assets/32621719/0cd2d7b4-f9b1-4320-821b-0868f57b5565)


BTW any ideas how to fix this better than just remove print syslog in qca-ssdk?

Reply:
> @robimarko @kirdesde I'm talking about these log entries: ![image](https://user-images.githubusercontent.com/32621719/280336330-0cd2d7b4-f9b1-4320-821b-0868f57b5565.png)
> 
> BTW any ideas how to fix this better than just remove print syslog in qca-ssdk?

I have the indoor version of this ap.

I have built with this PR included, and those spam messages are very annoying, they make the logs unuseable as they persist at a rate of 4 per second forever.

Reply:
I will add to that, since they are continually coming out of the serial console as well, the serial console is almost unuseable too.

Reply:
Yeah, I get the issue but just adding fake PHY-s is not the solution.
I will eventually look into the issue, maybe over the weekend to at least add a feature to disable the port if not used.